team-bayou/UNO-Shareboard
# Enhancement/#94/problem endpoint

To Test:
- Import postman collection
- Use the Submit Report endpoint under Report
- Try out different scenarios of data sending such as:
   > a proper bug and ad report
   > a improper bug report: Ex: leave out comments field to get a 400 BAD Request
   > a improper ad report: Leave out ad id or reporting user/ offending user to also get a 400
- To confirm that the proper report submits work correctly, check our email: unoshareboard.dev@gmail.com
